Client/Application/AccessibilityIdentifiers.swift

```diff
@@ -101,6 +101,11 @@ public struct AccessibilityIdentifiers {
         static let syncedTabs = "Synced Tabs"
         static let inactiveTabHeader = "InactiveTabs.header"
         static let inactiveTabDeleteButton = "InactiveTabs.deleteButton"
+        static let closeAllTabsButton = "closeAllTabsButtonTabTray"
+        static let newTabButton = "newTabButtonTabTray"
+        static let doneButton = "doneButtonTabTray"
+        static let syncTabsButton = "syncTabsButtonTabTray"
+        static let navBarSegmentedControl = "navBarTabTray"
```

Comment:
Just to note, if we add new AccessibilityIdentifers we should also replace them in tests classes where they are used. Just to make sure we're not in a weird state where our prod code use the identifiers but not the test code.